### PR TITLE
generate unique id using timestamp + random bits

### PR DESCRIPTION
**Pull request by SeeratSek**
- used rs/xid package (https://blog.kowalczyk.info/article/JyRZ/generating-good-unique-ids-in-go.html)
- change Players to use id as key
- players with same name can now play concurrently

### CODE REVIEW

File: server/models/player.go [hunk 1]
```diff
@@ -61,6 +64,11 @@ func CreatePlayer(n string, p Position, c string, ws *websocket.Conn) *Player {
 	}
 }
 
+func GenUniqueID() string {
+	id := xid.New()
```
Comment (teelau) [inline server/models/player.go]: cool new lib

Comment (brian-nguyen) [reply server/models/player.go]: Very cool indeed. Was this package added using dep? The Gopkg.toml file should have been updated

Comment (SeeratSek) [reply server/models/player.go]: soo I imported the package before calling dep -add and now i'm having trouble doing dep -add after the fact.. (https://github.com/golang/dep/issues/1670). I can just manually update the Gopkg.toml file with the constraint right?

Comment (brian-nguyen) [reply server/models/player.go]: Sure! Try that and when you update, I'll pull this branch and try running dep ensure to see if it works

**Pull request merged**